src/linear_samplers/linear_sampler_helpers/hadamard.jl

```diff
@@ -0,0 +1,46 @@
+"""
+fwht!(x::AbstractVector; signs=ones(Bool, size(x)), scaling = 1)  
+
+Performs an in-place Fast Walsh Hadamard Transform (FWHT). `signs` allows the user to input a boolean vector that can 
+flip the signs in the transform. `scaling` allows the user to scale the result of the transform. 
+
```

Comment:
Needs to be more detailed.

Reply:
I have attempted to add more details about this transform.